torch/_export/__init__.py

```diff
@@ -423,6 +427,88 @@ def _reorder_kwargs_by_names(arg_names: List[str], args: Tuple[Any], kwargs: Dic
     return OrderedDict({kw_name: kwargs[kw_name] for kw_name in arg_names[len(args):]})
 
 
+def save(
+    ep: ExportedProgram,
+    f: Union[str, pathlib.Path, io.BytesIO],
+    *,
+    extra_files: Optional[Dict[str, Any]] = None,
+    opset_version: Optional[Dict[str, int]] = None,
+) -> None:
+    """
+    Saves an ExportedProgram to a file-like object.
+
+    Args:
+        ep (ExportedProgram): The exported program to save.
+
+        f (str): A file-like object (has to implement write and flush)
+            or a string containing a file name.
+
+        extra_files (Optional[Dict[str, Any]]): Map from filename to contents
+            which will be stored as part of f.
+
+        opset_version (Optional[Dict[str, int]]): A map of opset names
+            to the version of this opset
+    """
+    from .serde.serialize import serialize
+    serialized_program, serialized_state_dict = serialize(ep, opset_version)
+
+    with _open_zipfile_writer(f) as zipf:
```

Comment:
Per discussion earlier, let's add a `version` file here, which stores an int. It represents versioning of both schema and file formats.